GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -42,23 +41,6 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
     } else {
```

Comment:
it is better practice to do:
```
if (!fallbackElement) {
    return self;
}
```
then you get less indentation and code is easier to read. The ruby guys called me on that one day while looking over my shoulder but I have also started to see it in other ios code bases